# Variant calling from NGS data of two accessions of Lablab purpureus

## Introduction

Lablab purpureus is a bean (family Fabaceae) commonly known as “lablab” which is native to Africa and widely cultivated in East Africa. It is called “Njahi” or black beans in Kenya where it is an important part of the daily diet. Lablab is, however, still an orphan crop with limited genomics and genetics resources. Recently, three genomes assemblies were separately created for two lablab accessions by the African Orphan Crops Consortium (AOCC) and ILRI (Feed and Forage Development program and Bioinformatics Community of Practice). These include an Illumina-based and an Oxford Nanopore-based assemblies for accession 147D, and a nanopore-based assembly for accession 147X. These assemblies can be used as templates for identifying molecular markers to fast-track the genetic mapping of economically important traits in lablab. 

## Objective
* To create a variant calling pipeline for Lablab purpureus.
* To convert the pipeline into a portable and reproducible workflow using Snakemake language.

## Data

Raw sequence data called sample X generated from minION and two reference files that is an Illumina-based and an Oxford Nanopore-based assemblies for accession 147D were provided.

## Pipeline

Snakemake workflow language was used as a 



## Contributors
* Immaculate Nahereza
* Jane Njeri
* Winfred Gatua 
* Nsangi Olga Tendo,
* Davis Kiberu 
* Nsubuga Moses 
* Eneza Yoel
